tests/ec2.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(feature = "ec2")]
+
+extern crate rusoto;
+
+use rusoto::ec2::{Ec2Client, DescribeInstancesRequest};
+use rusoto::{AwsError, ChainProvider, Region};
```

Comment:
The `AwsError` import is unused and should be removed.  This will cause a travisci build fail once https://github.com/rusoto/rusoto/pull/253 is merged in.